fhir-server-test/src/test/java/com/ibm/fhir/server/test/bulkdata/ExportOperationTest.java

```diff
@@ -349,8 +351,21 @@ public CloseableHttpClient getHttpClient() {
                 .build();
     }
 
-    public void verifyFileLines(String workItem, int count) throws IOException {
-        int actual = Files.readAllLines(new File(path + "/" + workItem).toPath()).size();
+    public void verifyFileLines(String workItem, int count, String resourceType) throws IOException {
+        List<String> lines = Files.readAllLines(new File(path + "/" + workItem).toPath());
```

Comment:
I wonder if we could just use Paths.get() instead
```suggestion
        List<String> lines = Files.readAllLines(Paths.get(path + "/" + workItem));
```